tools/cmake/executorch-config.cmake

```diff
@@ -213,3 +216,12 @@ foreach(lib ${backend_lib_list})
     target_link_options(executorch_backends INTERFACE ${lib})
   endif()
 endforeach()
+
+# An interface target containing all available extensions.
+add_library(executorch_extensions INTERFACE)
```

Comment:
how about also `add_library(executorch::extensions ALIAS executorch_extensions)`? also fine to skip for now and come back with a consistent namespacing strategy